src/binder/bind/bind_ddl.cpp

```diff
@@ -52,9 +54,20 @@ unique_ptr<BoundStatement> Binder::bindDropTable(const Statement& statement) {
     if (isNodeTable) {
         validateNodeTableHasNoEdge(catalog, tableID);
     }
-    return make_unique<BoundDropTable>(
-        isNodeTable ? (TableSchema*)catalogContent->getNodeTableSchema(tableID) :
-                      (TableSchema*)catalogContent->getRelTableSchema(tableID));
+    return make_unique<BoundDropTable>(tableID, tableName);
+}
+
+unique_ptr<BoundStatement> Binder::bindDropColumn(const Statement& statement) {
+    auto& dropColumn = (DropColumn&)statement;
+    auto tableName = dropColumn.getTableName();
+    validateTableExist(catalog, tableName);
+    auto catalogContent = catalog.getReadOnlyVersion();
+    auto isNodeTable = catalogContent->containNodeTable(tableName);
+    auto tableID = isNodeTable ? catalogContent->getNodeTableIDFromName(tableName) :
```

Comment:
Up to you but you can have an addition API in catalog getTableID(tableName) to which you push the check of isNodeTable.

Reply:
It is possible for a node table and a rel table to have the exactly the same name. So i can't create a generic getTableID(tableName) function